cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -1045,7 +1045,8 @@ def table_of_content(self, key, value):
     ).strip()
     if text != "--":
         chapters = re.split(r"; | -- |--", text)
-        return chapters
+        chapters = [elem.strip(' ') for elem in chapters]
+        return list(filter(None, chapters))
```

Comment:
Can it be empty string too (not only `None`) and you have to filter it out?

Reply:
for this you can use the decorator function, if I remember correctly it was `@out_strip`, automatically cleans the list of empty values

Reply:
No, with None we already filter the empty strings. With `None` we check if it's true or not for each element of the list and return the ones that are true, if an element is empty string it will be False and filtered out (that's why we apply the strip just above to remove any blank space on empty lines)